clean_up_dynamo/migrate_portfolios.py

```diff
@@ -0,0 +1,215 @@
+    json_record['GSI1SK'] = 'INTERNALITEM#' + format_key_value(internal_item_id)
+    json_record['imageUri'] = existing_item_json.get('image')
+    json_record['internalItemId'] = internal_item_id
+    json_record['itemType'] = 'internal'
+    json_record['PK'] = 'PORTFOLIO'
+    json_record['portfolioCollectionId'] = format_key_value(existing_item_json.get('collectionId'))
+    json_record['portfolioItemId'] = internal_item_id
+    json_record['portfolioUserId'] = user_id
+    json_record['sequence'] = int(existing_item_json.get('displayOrder', 0))
+    json_record['SK'] = 'USER#' + format_key_value(user_id) + '#' + format_key_value(existing_item_json.get('collectionId')) + '#' + format_key_value(internal_item_id)
+    json_record['title'] = existing_item_json.get('title')
+    json_record['TYPE'] = 'PortfolioItem'
+    json_record['uri'] = existing_item_json.get('manifest')
+    json_record['migratedDate'] = get_iso_date_as_string()
+    return json_record
+
+
+def create_json_records_for_collection(collection_uuid: str) -> list:
+    old_results = []
+    new_results = []
+    existing_collection_json = read_existing_collection(collection_uuid)  # could use same uuid value as collectionId
+    old_results.append(existing_collection_json)
+    user_uuid = existing_collection_json.get('userId')
+    existing_user_json = read_existing_user(user_uuid)
+    old_results.append(existing_user_json)
+    new_user_json = make_new_user_json(existing_user_json)  # good
+    new_results.append(new_user_json)
+    portfolio_user_id = new_user_json.get('portfolioUserId')
+    new_collection_json = make_new_collection_json(existing_collection_json, portfolio_user_id)
+    new_results.append(new_collection_json)
+    all_existing_items_json = read_all_existing_items_for_a_collection(collection_uuid)
+    for existing_item_json in all_existing_items_json:
+        old_results.append(existing_item_json)
+        new_item_json = make_new_item_json(existing_item_json, portfolio_user_id)
+        new_results.append(new_item_json)
+    # print('all_existing_items =', len(all_existing_items_json))
+    return new_results, old_results
+
+
+def capture_existing_collection_content(collections_of_interest_list: list, old_content_file_name: str, new_content_file_name: str):
+    all_new_results = []
+    all_old_results = []
+
+    for col in collections_of_interest_list:
+        new_results, old_results = create_json_records_for_collection(col)
+        all_new_results.extend(new_results)
+        all_old_results.extend(old_results)
+
+    with open(new_content_file_name, 'w') as output_file:
+        json.dump(all_new_results, output_file, indent=2)
+
+    with open(old_content_file_name, 'w') as output_file:
+        json.dump(all_old_results, output_file, indent=2, default=str)
+
+
+def save_collection_content_to_new_table(table_name: str, new_content_file_name: str):
+    with io.open(new_content_file_name, 'r', encoding='utf-8') as json_file:
```

Comment:
remove the "import io" at the top of this module and change to

```suggestion
    with open(new_content_file_name, 'r', encoding='utf-8') as json_file:
```